cime_config/config_compsets.xml

```diff
@@ -266,6 +265,7 @@
         <value grid="a%0.9x1.25_l%0.9x1.25_oi%gx1v7_r%r05_g%gland4_w%ww3a_m%gx1v7"      compset="1850_CAM60%WC.*_CLM50%BGC-CROP.*_CICE.*_POP2%ECO.*_MOSART_CISM2%NOEVOLVE_WW3">0134-01-01</value>
         <value grid="a%0.9x1.25_l%0.9x1.25_oi%gx1v7_r%r05_g%gland4_w%ww3a_m%gx1v7"      compset="HIST_CAM60%WC.*_CLM50%BGC-CROP.*_CICE.*_POP2%ECO.*_MOSART_CISM2%NOEVOLVE_WW3">0056-01-01</value>
         <value grid="a%0.9x1.25_l%0.9x1.25_oi%gx1v7_r%r05_g%gland4_w%ww3a_m%gx1v7"      compset="SSP(126|245|370|585)_CAM60%WCTS_CLM50%BGC-CROP-CMIP6WACCMDECK_CICE%CMIP6_POP2%ECO%NDEP_MOSART_CISM2%NOEVOLVE_WW3">2015-01-01</value>
+        <value grid="a%0.9x1.25_l%0.9x1.25_oi%gx1v7_r%r05_g%gland4_w%ww3a_m%gx1v7"      compset="^SSP[0-9]_CAM60_CLM50%BGC-CROP.*CICE.*POP2%ECO.*_MOSART_CISM2%NOEVOLVE_WW3">2015-01-01</value>
```

Comment:
@fvitt and @fischer-ncar we are currently using two or three lines for the REFCASES for the SSP's. For some of these things we could consolidate it into a single line, if a SSP is being run, set it to hybrid, with cesm2_init, and 2015-01-01 start. The only one we need three lines on is what REFCASE is used for a particular one.

I think this would be worth doing, do we all agree on that? If so I could do a change, try it out, and make sure we agree on how it should look. Let me know what you think...

Reply:
This comment is actually outdated now. The most recent version uses SSP[0-9]+ in most places, and the explicit SSP(126|245|370|585)_ only to set the WACCM specific REFCASES. That's the case that needs three lines, one for standard WACCM, and one for overshoot WACCM, and then the catch all for non-WACCM SSP's. So I think this is OK.

Does that sound good to you?